keps/sig-contributor-experience/0005-community-site.md

```diff
@@ -60,30 +60,47 @@ Specifically, building a site that can take advantage of the KEP metadata will b
 
 ### Goals
 
-* Provide a new site (`community.kubernetes.io`? `contrib.kubernetes.io`?).
-* Publish information that is currently in the [community repo](https://github.com/kubernetes/community).
+* A community facing portal to collect information for those actively working on upstream Kubernetes.
+* An easy to remember URL. (`community.kubernetes.io`? `contrib.kubernetes.io`?)
+* A streamlined process to update and share this information.
+  Ownership should be delegated using the existing OWNERS mechanisms.
+* A site that will be indexed well on Google to collect markdown files from the smattering of repos that we currently have.
+  This includes information that is currently in the [community repo](https://github.com/kubernetes/community).
+* Provide a place to launch and quickly evolve the contributor handbook.
 * Build some simple tools to enhance discoverability within the site.
   This could include features such as automatically linking KEP and SIG names.
 * Over time, add an index of events, meetups, and other forums for those that are actively contributing to k8s.
 
 ### Non-Goals
 
-* Discover and bring together information from multiple orgs/repos.
+* Actively migrate information from multiple orgs/repos.
+  This should be a place that people in the community choose to use to communicate vs. being forced.
```

Comment:
I agree about not forcing, and creating a place where people choose to communicate. But couldn't we also encourage folks to migrate information from other repositories? There's a huge information architecture issue here, and without wanting to impose requirements on anyone, using the site as an opportunity to organize content better could be a huge win.